esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -24,15 +24,26 @@
       grid (you can use the preprocessor :func:`esmvalcore.preprocessor.regrid`
       for this). Input data needs to be 2D with dimensions `latitude`,
       `longitude`.
-    - Vertical profiles (plot type ``profile``): for each variable and dataset,
-      an individual profile is plotted. If a reference dataset is defined, also
-      include this dataset and a bias plot into the figure. Note that if a
-      reference dataset is defined, all input datasets need to be given on the
-      same horizontal and vertical grid (you can use the preprocessors
-      :func:`esmvalcore.preprocessor.regrid` and
+    - Zonal mean profiles (plot type ``zonal_mean_profile``):
+      for each variable and dataset, an individual profile is plotted. If a
+      reference dataset is defined, also include this dataset and a bias plot
+      into the figure. Note that if a reference dataset is defined, all input
+      datasets need to be given on the same horizontal and vertical grid (you
+      can use the preprocessors :func:`esmvalcore.preprocessor.regrid` and
       :func:`esmvalcore.preprocessor.extract_levels` for this). Input data
       needs to be 2D with dimensions `latitude`, `height`/`air_pressure`.
 
+      .. warning::
+
+      The plot_type ``profile`` for zonal mean profiles has been deprecated
+      in ESMValTool version 2.9.0 and is scheduled for removal in version
+      2.11.0. Please use plot type ``zonal_mean_profile`` instead. This is
+      an exact replacement.
```

Comment:
```suggestion
          The plot_type ``profile`` for zonal mean profiles has been deprecated
          in ESMValTool version 2.9.0 and is scheduled for removal in version
          2.11.0. Please use plot type ``zonal_mean_profile`` instead. This is
          an exact replacement.
```

This should fix the documentation build